doctr/local.py

```diff
@@ -39,10 +40,10 @@ def encrypt_variable(variable, build_repo, public_key=None):
         raise ValueError("variable should be of the form 'VARIABLE=value'")
 
     if not public_key:
-        # TODO: Error handling
         r = requests.get('https://api.travis-ci.org/repos/{build_repo}/key'.format(build_repo=build_repo),
             headers={'Accept': 'application/vnd.travis-ci.2+json'})
-        r.raise_for_status()
```

Comment:
It's still good to have this in case of other errors (though catching it and printing the error can be fine).